frontend/packages/data-portal/app/components/Dataset/RunsTable.tsx

```diff
@@ -219,8 +219,8 @@ export function RunsTable() {
               <ViewTomogramButton
                 tomogramId={tomogram?.id.toString()}
                 buttonProps={{
-                  sdsType: 'secondary',
-                  sdsStyle: 'square',
+                  sdsType: 'primary',
+                  sdsStyle: 'minimal',
```

Comment:
@kev-zunshiwang see here for the sds settings of the minimal button. I will investigate the hover color. Thanks!